app/web/features/auth/signup/Signup.tsx

```diff
@@ -126,6 +126,14 @@ function CurrentForm() {
         <Typography variant="h1" gutterBottom>
           {t("auth:feedback_form.header")}
         </Typography>
+        <Trans i18nKey="auth:feedback_form.skiptitle">
+        Not interested? Click 
+            <StyledLink href={loginRoute}>
+            here
+            </StyledLink>
+            to skip this step.
+          </Trans>
+        </Typography>
```

Comment:
I think you've missed an opening `<Typography>` here - the build choked on some syntax error.

Also, I think it's better to wrap the "skip this step" as the link and remove "click here" in the text as that's quite poor for accessibility. Not really your fault as I know you were following the designs, but probably the better thing to do when you get around changing things